scijava/scijava-ops-engine/src/test/java/org/scijava/ops/engine/impl/ProvenanceTest.java

```diff
@@ -1,11 +1,7 @@
 
 package org.scijava.ops.engine.impl;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.List;
+import java.util.*;
```

Comment:
Since when did we decide to start using wildcard imports? I don't remember discussing this.